### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by vipchristian** [Codeinwp/otter-blocks#2280]
### Description

Some HTML codes fail Chrome's accessibility test. This will be mandatory by law (BFSG) in Germany from 2025.
For example:
/wp-content/plugins/otter-blocks/build/blocks/slider.js
The arrows on the slide have no aria-label="name”

Other blocks such as Accordeon also have accessibility problems.

Thx guys!

### Step-by-step reproduction instructions

Go to https://demo.themeisle.com/otter-blocks/slider/
Chrome -> Inspect -> Lighthouse:
Device: Desktop
Categories: Accessibility

Result: "Buttons do not have an accessible name"

### Is the issue you are reporting a regression

No

**Comment by Captain-Arnab** [comment]
After thoroughly studying the bug I came to an conclusion that it needs to be fixed in slider.js by replacing 


original code :
`<button class="glide__arrow glide__arrow--right" data-glide-dir="&gt;">`

New code : 
`<button class="glide__arrow glide__arrow--right" data-glide-dir="&gt;" aria-label="Next Slide">
`

Same as for left button


I hope it will work
